tox.ini

```diff
@@ -36,6 +36,7 @@ deps =
     xunitparser
     pytest
     pytest-cov
+    cocotb-bus
```

Comment:
I think this line is a mistake

Reply:
I understand tox doesn't separate build dependencies from runtime dependencies, so it installes depends first. We have a catch-22. Should cocotb-bus it be installed via extras?

Reply:
Is this actually a dependency anyway? Surely its only needed for the docs?

Reply:
A lot of tests depend on what was moved. I just tested it, using `extras` works. I will also add `pip_pre = true` option and we can "yank" cocotb-bus v0.1.1 from Pypi and revert that commit, if desired.